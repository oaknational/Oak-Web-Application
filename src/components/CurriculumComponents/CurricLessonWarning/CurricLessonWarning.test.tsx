import { CurricLessonWarning } from "./";

import renderWithTheme from "@/__tests__/__helpers__/renderWithTheme";

describe("CurricLessonWarning", () => {
  test("basic usage", () => {
    const { baseElement } = renderWithTheme(
      <CurricLessonWarning count={4} total={6} />,
    );
    expect(baseElement).toHaveTextContent("4/6 lessons");
    expect(baseElement).toMatchSnapshot();
  });
});
