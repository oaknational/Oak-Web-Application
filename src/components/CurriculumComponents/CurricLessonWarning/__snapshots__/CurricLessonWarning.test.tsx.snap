// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurricLessonWarning basic usage 1`] = `
<body>
  <div>
    <div
      class="sc-fqkvVR sc-kAyceB fgBWrW fEAjEF"
    >
      <div
        class="sc-fqkvVR sc-kAyceB ldgGpl joHkzz"
      />
       
      4
      /
      6
       lessons
    </div>
  </div>
</body>
`;
